pypeit/utils.py

```diff
@@ -1790,24 +1790,22 @@ def is_float(s):
 
 
 def find_single_file(file_pattern) -> pathlib.Path:
-    """Find a single file matching a wildcard pattern.
+    """
+    Find a single file matching a wildcard pattern.
 
     Args:
-        file_pattern (str): A filename pattern, see the python 'glob' module.
+        file_pattern (str):
+            A filename pattern, see the python 'glob' module.
 
     Returns:
-        :obj:`pathlib.Path`: A file name, or None if no filename was found. This will give a warning
-             if multiple files are found and return the first one.
+        :obj:`pathlib.Path`: A file name, or None if no filename was found. This
+        will give a warning if multiple files are found and return the first
+        one.
     """
-
     files = glob.glob(file_pattern)
```

Comment:
Not sure if it matters, but if we sort this list, there might be a consistent pattern to which "first file" is returned in the event of multiple matches.

Reply:
Very good point!  Done!